hopli/src/faucet.rs

```diff
@@ -28,10 +25,7 @@ pub struct FaucetArgs {
     address: Option<String>,
 
     #[clap(flatten)]
-    password: PasswordArgs,
-
-    #[clap(flatten)]
-    local_identity: LocalIdentityArgs,
+    local_identity: IdentityFileArgs,
```

Comment:
Wait, why's there multiple clap objects for a single binary?

Reply:
There are several subcommands, if this is what you referred to?